schedule/yast/sle/flows/default_s390x_kvm.yaml

```diff
@@ -33,7 +33,8 @@ software: []
 booting:
   - installation/bootloader_settings/disable_boot_menu_timeout
 security: []
-default_systemd_target: []
+default_systemd_target:
+  - installation/installation_settings/validate_default_target
```

Comment:
The idea was not to put this line in the default file. The default should still provide that empty hook. The idea was to use this hook in your test yaml schedule.

Reply:
Modifiled.